### DISCUSSION TITLE
Не могу запустить

### DISCUSSION THREAD

**Discussion by shmuz** [elfmz/far2l#1142] [Q&A] [answered]
Собрал сабж (cmake, затем make), сообщений об ошибках не было.
Далее пытаюсь запустить `far2l` - выдаёт `far2l: command not found`.
Нужна помощь.

**Comment by Katarn** [comment, marked as answer]
`./far2l` из каталога с собранным бинарником.

**Comment by shmuz** [comment]
Большое спасибо! 
Нашёл и запустил - бинарник оказался в папке `_build/install`.
Почему-то команда `whereis far2l` его не находила.
